src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -8344,23 +8308,19 @@ namespace AirflowNetwork {
         int LT;
         int CompNum;
         iComponentTypeNum CompTypeNum;
-        int TypeNum;
-        std::string CompName;
         Real64 DirSign;
         Real64 COZN;
         int ZoneNum;
-        bool found;
-        bool OANode;
 
         MA = 0.0;
         MV = 0.0;
         for (i = 1; i <= AirflowNetworkNumOfLinks; ++i) {
             CompNum = AirflowNetworkLinkageData(i).CompNum;
             CompTypeNum = AirflowNetworkCompData(CompNum).CompTypeNum;
-            CompName = AirflowNetworkCompData(CompNum).EPlusName;
+            std::string CompName = AirflowNetworkCompData(CompNum).EPlusName;
             // Calculate duct moisture diffusion loss
             if (CompTypeNum == iComponentTypeNum::DWC && CompName == std::string()) { // Duct component only
```

Comment:
Use `CompName.empty()` instead of constructing an empty string and comparing to it.